src/client/pages/Embark/index.tsx

```diff
@@ -171,17 +184,7 @@ const Embark = (props: EmbarkProps) => {
                 passage={currentPassage}
                 storyData={state.data}
                 startPageLink={props.startPageLink}
-                customTrailingContent={
```

Comment:
So, by design, there shouldn't be a case where both `LanguagePicker` and `CallCenterPhoneNumber` should be displayed?! 🤔

Reply:
Exactly, they occupy the same space on the page